projects/igniteui-angular/src/lib/drop-down/drop-down.component.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { Component, ViewChild, OnInit, ElementRef, ViewChildren, QueryList } from '@angular/core';
+import { Component, ViewChild, OnInit, ElementRef, ViewChildren, QueryList, IterableChanges } from '@angular/core';
```

Comment:
The imported `IterableChanges` don't seem to be used. If so, please remove them.